packages/wonder-blocks-color/src/__docs__/color.stories.mdx

```diff
@@ -0,0 +1,234 @@
+    offBlack64: Color.offBlack64,
+    offBlack50: Color.offBlack50,
+    offBlack32: Color.offBlack32,
+    offBlack16: Color.offBlack16,
+};
+
+export const WhiteShades = {
+    offBlack8: Color.offBlack8,
+    offWhite: Color.offWhite,
+    offWhite64: Color.offWhite64,
+    offWhite50: Color.offWhite50,
+    white: Color.white,
+};
+
+export const BrandColors = {
+    darkBlue: Color.darkBlue,
+    lightBlue: Color.lightBlue,
+    teal: Color.teal,
+    pink: Color.pink,
+};
+
+# Color
+
+<Subtitle>
+    <ComponentInfo name={name} version={version} />
+</Subtitle>
+
+## Summary
+
+<ColorPalette>
+    <ColorItem
+        title="Product colors"
+        subtitle="Use only accessible colors against appropriate backgrounds for links, buttons, and other navigational elements."
+        colors={ProductColors}
+    />
+    <ColorItem
+        title="Neutral colors"
+        subtitle="Used for text and chrome components."
+        colors={NeutralColors}
+    />
+    <ColorItem colors={WhiteShades} />
+    <ColorItem
+        title="Brand colors"
+        subtitle="Use for designing anything centered around Khan Academy as a brand. Secondary brand colors should only be used for decoration. Expanded brand palette used for illustrations."
+        colors={BrandColors}
+    />
+</ColorPalette>
+
+## Usage
+
+`wonder-blocks-color` exports a set of colors that can be used in your
+components. The colors are exported as constants and can be used like so:
+
+
+```js
+import Color from "@khanacademy/wonder-blocks-color";
+const styles = {background: Color.darkBlue};
+```
+
+### Product
+
+Use only accessible colors against appropriate backgrounds for links, buttons,
+and other navigational elements.
+
+<Swatch
+    name="blue"
+    color={Color.blue}
+    use="text"
+    description="Actionable. Used to indicate actionable items like links and buttons."
+/>
+
+<Swatch
+    name="purple"
+    color={Color.purple}
+    use="icons"
+    description="Used to indicate Mastery."
+/>
+
+<Swatch
+    name="gold"
+    color={Color.gold}
+    description="Used as an accent for celebratory moments. If using for icons, accompany with descriptive text."
+    segments={1}
+/>
+
+<Swatch
+    name="green"
+    color={Color.green}
+    use="icons"
+    description="Affirmative. Used to indicate success states and correctness. Do not use for text."
+/>
+
+<Swatch
+    name="red"
+    color={Color.red}
+    use="text"
+    description="Negative. Used for errors and destructive actions."
+/>
+
+---
+
+### Neutral
+
+Used for text and chrome components.
+
+<Swatch
+    name="offBlack"
+    color={Color.offBlack}
+    use="text"
+    description="Primary text color. Can also used for links in lists."
+/>
+
+<Swatch
+    name="offBlack64"
+    color={Color.offBlack64}
+    use="text"
+    description="Secondary text color and modal backdrop color."
+/>
+
+<Swatch
+    name="offBlack50"
+    color={Color.offBlack50}
+    description="Fieldset borders"
```

Comment:
**Super duper nit:** Can you add periods to the end of all of these descriptions for consistency